net/local/local_connect.c

```diff
@@ -238,7 +238,9 @@ int psock_local_connect(FAR struct socket *psock,
 {
   FAR struct local_conn_s *client;
   FAR struct sockaddr_un *unaddr = (FAR struct sockaddr_un *)addr;
+  FAR const char *unpath = unaddr->sun_path;
   FAR struct local_conn_s *conn = NULL;
```

Comment:
Not related to this PR, but seems that `NULL` init for `conn` is not needed.

Reply:
@pkarashchenko is alls a good practice to NULL initialize pointers to avoid Dangling Pointer? https://en.wikipedia.org/wiki/Dangling_pointer

Reply:
Yes, but in such way the C standard would always init stack variables :)
The init is needed if pointer is not unconditionally overwritten. In this case the init  is followed by explicit overwrite in `while ((conn = local_nextconn(conn)) != NULL)`, so initing is only wasting CPU cycles.